app/src/main/java/org/piramalswasthya/sakhi/utils/KeyUtils.kt

```
@@ -0,0 +1,35 @@
+package org.piramalswasthya.sakhi.utils
+
+
+object KeyUtils {
+
+    private const val NATIVE_JNI_LIB_NAME = "sakhi"
+
+    init {
+        try {
+            System.loadLibrary(NATIVE_JNI_LIB_NAME)
+        } catch (e: UnsatisfiedLinkError) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)